### ISSUE TITLE
Help wanted: Looking for maintainers

### ISSUE THREAD

**Issue by christianvoigt** [christianvoigt/language-argdown#13]
When we started out developing Argdown we thought we would use Atom as an editor. So I started this repository to try out the syntax highlighting. We then switched to VSCode and abandoned this repository. At the moment I have not enough time to maintain both repositories.

However, there still seem to be users who would like to use Argdown in Atom, so it would be great if some of you would take over this repository and bring the syntax highlighting up to date. This plugin uses a [Textmate grammar file](https://flight-manual.atom.io/hacking-atom/sections/creating-a-legacy-textmate-grammar/). The bad news is that Atom prefers [TreeSitter grammars](https://flight-manual.atom.io/hacking-atom/sections/creating-a-grammar/) by now. 

The good news is that Textmate grammars are still supported and it would be very easy to copy & paste the VSCode [tmLanguage definition](https://github.com/christianvoigt/argdown/blob/master/packages/argdown-vscode/syntaxes/argdown.tmLanguage.json) over to this repository and convert it from json to cson. @nathan-artist maybe you might be interested in this?

Obviously it would be even more amazing if someone would write a TreeSitter grammar for Argdown! That would be useful in many other ways as well.

**Comment by rapus95** [comment]
I guess, if you have or build a language server (for vs-code of course) you could just backend the same language server into Atom. Which would drastically reduce the time to maintain the extension.

**Comment by christianvoigt** [comment]
yes, we already have one, which could be used: https://github.com/christianvoigt/argdown/tree/master/packages/argdown-language-server

The syntax highlighting is not part of it though and has to be maintained separately.

**Comment by nathan-artist** [comment]
@christianvoigt I use Argdown in Atom so I may be willing to maintain this repository, though not right away. At some point I will take a look at converting the VSCode tmLanguage definition that you mentioned above, and if I am able to do it successfully (which seems likely) then I can start maintaining the repository from that point forward.

Unfortunately, writing a TreeSitter grammar likely requires more JavaScript knowledge than I currently command.

**Comment by rapus95** [comment]
Well, Atom has legacy support for textmate grammars IIRC, and it seems you can also convert them: https://flight-manual.atom.io/hacking-atom/sections/converting-from-textmate/

**Comment by nathan-artist** [comment]
@rapus95 Thanks, yes, that would be my plan.

**Comment by christianvoigt** [comment]
Hi @nathan-artist, sorry for not replying earlier. We moved to a new apartment in January and I have neglected Argdown for the last two months. I will get back to it in the following weeks. Let me know if you are making any progress on the tmLanguage conversion or if you need any help with it.

I am really glad that you are interested in maintaining this repository!

**Comment by nathan-artist** [comment]
@christianvoigt I have not looked at it yet. I will let you know when I start. If anyone else wants to start before I do, please feel free.

**Comment by RedCMD** [comment]
here's my VSCode extension to help with authoring TextMate grammars https://marketplace.visualstudio.com/items?itemName=RedCMD.tmlanguage-syntax-highlighter
